src/Raven.Studio/typescript/components/pages/database/settings/documentRefresh/DocumentRefresh.tsx

```diff
@@ -0,0 +1,198 @@
+                                </Card>
+                            </Col>
+                        </Form>
+                    </Col>
+                    <Col sm={12} lg={4}>
+                        <AboutViewAnchored>
+                            <AccordionItemWrapper
+                                targetId="1"
+                                icon="about"
+                                color="info"
+                                description="Get additional info on what this feature can offer you"
+                                heading="About this view"
+                            >
+                                <p>
+                                    When <strong>Document Refresh</strong> is enabled:
+                                </p>
+                                <ul>
+                                    <li>
+                                        The server scans the database at the specified <strong>frequency</strong>, searching for documents that should be refreshed.
+                                    </li>
+                                    <li>
+                                        Any document that has a <code>@refresh</code> metadata property whose time has passed at the time of the scan will be modified by removing this property.
+                                    </li>
+                                    <li>
+                                        This modification will trigger any processes related to the document, such as: re-indexing or taking part in an ongoing-task (e.g. Replication, ETL, Subscriptions, etc.), as defined by your configuration.
+                                    </li>
+                                </ul>
+                                <p>Sample document:</p>
+                                <Code code={codeExample} language="javascript"/>
+                                <hr />
+                                <div className="small-label mb-2">useful links</div>
+                                <a href="https://ravendb.net/l/1PKUYJ/6.0/Csharp" target="_blank">
+                                    <Icon icon="newtab" /> Docs - Document Refresh
+                                </a>
+                            </AccordionItemWrapper>
+                        </AboutViewAnchored>
+                    </Col>
+                </Row>
+            </Col>
+        </div>
+    );
+}
+
+function mapToFormData(dto: ServerRefreshConfiguration): DocumentRefreshFormData {
+    if (!dto) {
+        return {
+            isDocumentRefreshEnabled: false,
+            isRefreshFrequencyEnabled: false,
+            refreshFrequency: null,
+        };
+    }
+
+    return {
+        isDocumentRefreshEnabled: !dto.Disabled,
+        isRefreshFrequencyEnabled: dto.RefreshFrequencyInSec != null,
+        refreshFrequency: dto.RefreshFrequencyInSec,
+    };
+}
+
+const codeExample = `
+{
+    "Example": "Set a timestamp in the @refresh metadata property",
```

Comment:
Seems this causes the bottom scroll bar to appear
Suggesting this shorter text instead:
```"Example": "Set timestamp in the @refresh metadata property",```

Reply:
We'll handle that within [RavenDB-20894](https://issues.hibernatingrhinos.com/issue/RavenDB-20894/Custom-scrollbar-for-React-components)